src/java.base/share/classes/java/nio/file/attribute/UserPrincipalNotFoundException.java

```diff
@@ -40,6 +40,9 @@
     @java.io.Serial
     static final long serialVersionUID = -5369283889045833024L;
 
+    /**
+     * The name of the {@code UserPrincipal} that does not exist.
```

Comment:
Probably best to use "user principal name" so that it is consistent with getName.

Reply:
Thanks Alan, will update as you suggest shortly for both your comments.